doc/rst/language/spec/modules.rst

```diff
@@ -273,9 +273,26 @@ Visibility Of A Module
 ~~~~~~~~~~~~~~~~~~~~~~
 
 A top-level module is available for use (:ref:`Using_Modules`) or
-import (:ref:`Importing_Modules`) anywhere. The visibility of a nested module is
-subject to the rules of :ref:`Visibility_Of_Symbols`, where the nested module is
-considered a "module-scope symbol" of its outer module.
+import (:ref:`Importing_Modules`) anywhere.
+
+For the purpose of ``use``, the visibility of a nested module is subject to the
+rules of :ref:`Visibility_Of_Symbols`, where the nested module is considered a
+"module-scope symbol" of its outer module.  If the module is currently in scope,
+then it may be used with just its name.  The module may alternatively be
+accessed explicitly with all the outer modules surrounding it to the top level,
+or relatively from the current scope with ``this`` or ``super`` components.
+Which of ``this`` or ``super`` is appropriate depends on its placement relative
+to the current scope - if the submodule is within the current scope, then
+``this`` is used; if the submodule is within a parent module's scope, then one
+or more ``super`` components can be used.
+
+For the purpose of ``import``, the module is known to be defined at the same
+location as for ``use`` statements.  However, in order to ``import`` it, either
+all the outer modules surrounding it to the top level must be provided as part
+of the path to the module, or a ``super`` or ``this`` prefix may be provided.
+The module may not be imported with just its name, even from the scope in which
```

Comment:
Maybe you mean this paragraph to be about submodules or something? This sentence seems false to me (we can import top level modules just with their name).

Reply:
Yeah, I was trying to indicate that by calling back to the just previous paragraph.